heltour/tournament/lichessapi.py

```diff
@@ -8,9 +8,15 @@
 logger = logging.getLogger(__name__)
 
 
-def _apicall(url, timeout=1800, check_interval=0.1, post_data=None):
+def _apicall(
+    url: str,
+    timeout: int = 1800,
+    check_interval: float = 0.1,
+    post_data: str = "",
+    post: bool = False
```

Comment:
This should probably just be an enum for all of the different METHOD's that we might use